_raml/apps/products/examples/patch_product_productid_request.json

```diff
@@ -12,4 +12,27 @@
   "op": "add",
   "path": "/deliveryWeight",
   "value": {"unit":"g", "amount": 260}
-}]
+},
+{
+  "op": "add",
+  "path": "/customerSpecificPriceInfo",
+  "value": [{"customerGroup":"KG1",
+                "priceInfo": [
+                  {
+                    "quantity" : 4,
+                    "price" : 222.00,
+                    "discountPercentage" : 20 
+                  },
+                  {
+                    "quantity" : 9,
+                    "price" : 100.00,
+                    "discountPercentage" : 10 
+                  },
+                  {
+                    "quantity" : 55,
+                    "price" : 100.00,
```

Comment:
@mhoehn I'm wondering if it makes sense that the price for 55 pieces is the same as for 9. The quantity refers to the same product, correct? So if a customer orders 55 instead of just 9 pieces, the price should be way lower, shouldn't it?